core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/ConfigDoItemFinder.java

```diff
@@ -67,12 +69,13 @@ ScannedConfigDocsItemHolder findInMemoryConfigurationItems() {
      * Recursively find config item found in a config root given as {@link Element}
      *
      * @param element - root element
-     * @param parentName - root name
+     * @param rootName - root name
+     * @param parentName - parent name
      * @param configPhase - configuration phase see {@link ConfigPhase}
      * @param withinAMap - indicates if a a key is within a map or is a map configuration key
      * @param sectionLevel - section sectionLevel
      */
-    private List<ConfigDocItem> recursivelyFindConfigItems(Element element, String parentName,
+    private List<ConfigDocItem> recursivelyFindConfigItems(Element element, String rootName, String parentName,
```

Comment:
When would `rootName` be different than `parentName`?

Reply:
When scanning group configs items -see  https://github.com/quarkusio/quarkus/pull/8183/files#diff-5b3aaa451926ba3ee4ee4207f00433f1R289

So taking the `liveness-probe` config group for example, we'll have `kubernetes` as root name,  and `kubernetes.liveness-probe` as parent name.

Reply:
That's to say, the `rootName` does not change with depth while the `parentName` changes.

Reply:
Gotcha, thanks!